### DISCUSSION TITLE
Copilot skittish around the word trans[late,lation,fer]

### DISCUSSION THREAD

**Discussion by sudofox** [community/community#19381] [Copilot Conversations]
github copilot is super skittish when doing localization-related things -- I have a suspicion why: think it might be refusing to autocomplete anything that could have the word "trans" in it for some reason. for example, with many other examples in the document of it and an obvious next thing, it just won't autocomplete, for example:

`<textarea aria-label="{{ "VIEWPAGE_COMMENT_TEXTBOX" `

which would obviously be completed as ` | trans }}"`

Other examples:

```
{{ "USERS" | trans }}
{{ "SEARCH_RESULTS" | trans }}
```
or
```
{% trans "SEARCH_RESULTS" %}
```
I've had this issue for a while and it really makes localizing my webpages a slog -- been trying to make the site more accessible to people.

Also seems to dislike "translate", "transfer", and "translation", it just gets hung up on there.

**Comment by sudofox** [comment]
This is _really, really_ annoying when writing twig templates.

**Comment by RR-28023** [comment]
Yes! And when working with the SMPL models (which have a `trans` parameter, standing for "translation")
